pkg/specgen/specgen.go

```diff
@@ -44,7 +44,6 @@ type ContainerBasicConfig struct {
 	Command []string `json:"command,omitempty"`
 	// Env is a set of environment variables that will be set in the
 	// container.
-	// Optional.
```

Comment:
I'm pretty sure this is still Optional - the user doesn't need to specify any environment variables to successfully make a container.